csrc/backend_ops/tensorrt/multi_level_roi_align/trt_multi_level_roi_align_kernel.cu

```diff
@@ -62,18 +58,21 @@ __device__ scalar_t roi_align_single(const scalar_t *bottom_data, const int roi_
   return output_val;
 }
 
-template <typename scalar_t>
-__global__ void roi_extractor_kernel(scalar_t *output, const scalar_t *bottom_rois,
-                                     FeatData feat_data, const int sample_num,
-                                     const float roi_scale_factor, const int finest_scale,
-                                     const int pooled_height, const int pooled_width,
-                                     const bool aligned, int nThreads) {
+template <typename scalar_t, bool aligned>
```

Comment:
What's the necessity of adding a non-type template parameter `aligned`?

Reply:
I guess there would be less jump if aligned is constant. Which would benefit block level optimize.